_notices/rgn0013.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 13 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "RAPIDS switching to CalVer in v0.20"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Release Version Change
+notice_rapids_version: "v0.20"
+notice_created: 2021-05-05
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2021-05-05
+---
+
+## Overview
+
+All RAPIDS projects are switching versioning scheme to [Calendar Versioning](https://calver.org).
+
+## Impact
+
+Instead of a `0.20.0` release in June 2021, we will release `21.06.00`
+
+New version format for release: `YY.MM.PP`
+New version format for nightlies: `YY.MM.PPaYYMMDD`
+Where `PP` is the two digit patch number not day.
```

Comment:
I see these are on separate lines in the markdown file, but they're appearing all on a single line on the rendered page. You may need to insert an extra line break between these lines if you want them to display in separate `<p>` tags (and therefore separate lines) on the rendered HTML page. Alternatively, you could add some punctuation if you want them to all appear on a single line.

![image](https://user-images.githubusercontent.com/7400326/117165070-2c4d6700-ad93-11eb-8a25-1ca134354a08.png)